### PR TITLE
[wct] Updated Mocha to the latest version and regenerated a lockfile.

### LINKED ISSUE #227: [wct] Update Mocha
We should update Mocha version, since it resolves multiple bugs and introduces some new features. It also contains some braking changes, so it requires a major version bump of WCT. Please consult Mocha changelog to learn more.

### PR DESCRIPTION
**Pull request by merlinnot**
Tests and build are passing locally.

Resolves #227

### CODE REVIEW

File: packages/web-component-tester/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 <!-- ## Unreleased -->
 <!-- Add new, unreleased items here. -->
+### Security
```
Comment (rictic) [inline packages/web-component-tester/CHANGELOG.md]: Preferred format is:

```
## Unreleased
 * Updated Mocha to v5
```

**Comment by merlinnot** [comment]
Thank you for a review. Yes, this is separate from `wct-browser-legacy`.

**Comment by merlinnot** [comment]
@rictic a friendly ping :)

**Comment by rictic** [comment]
LGTM!

**Pull request merged**